test/commentsCounter.test.js

```diff
@@ -0,0 +1,12 @@
+/** @jest-environment jsdom */
+import commentsCounter from '../src/modules/commentsCounter.js';
+
+describe('commentsCounter,', () => {
+  document.body.innerHTML = '<span id="c-total"></span>';
+  const count = document.getElementById('c-total');
+  const comments = [{ username: 's', comment: 'd', creation_date: '2023-02-17' }];
+  commentsCounter(comments);
+  test('displays the number of comments', () => {
+    expect(count.textContent).toBe('1');
+  });
+});
```

Comment:
- [ ] Great Job Creating tests for both like & comment counters, but as part of the requirements you need to test all the edge cases as well, kindly add at least 1 edge case example : `the scenario in which there are 0 items on the page (counter function should return 0)`
kindly add an edge case for both Counters Comments & Like 🙏